pkger/parser.go

```diff
@@ -746,13 +746,29 @@ func parseChart(r Resource) (chart, []ValidationErr) {
 		c.Axes = presAxes
 	} else {
 		for _, ra := range r.slcResource(fieldChartAxes) {
+			domain := []float64{}
```

Comment:
nit: can you make this `var domain []float64`, the behavior when encoding could make this empty slice appear valid instead of it not being there. The `var domain []float64` will be converted to a null to JSON, when you have `domain := []float64{}`, it is converted into an empty slice. Generally you'll want to prefer the `var` approach in pkger, unless the `[]` is desired, which it is in some cases (UI may require it).

Reply:
good deal